src/checkPassword.test.js

```diff
@@ -7,13 +7,52 @@ describe(`Function 'checkPassword':`, () => {
     expect(checkPassword).toBeInstanceOf(Function);
   });
 
+  it(`Should accept a string value`, () => {
+    const password = 'Password1&';
+
+    checkPassword(password);
+
+    const typeInputValue = typeof password;
+
+    expect(typeInputValue).toBe('string');
+  });
+
   it(`should return boolean`, () => {
```

Comment:
This test case is also unnecessary, as it's just checking the return type, which is already implied by other test cases. Please remove this test case.